ObsCoreExtensionForRadioData.tex

```diff
@@ -338,86 +330,112 @@ \subsection{uv parameters}
 
 
-\section{How to implement the extension in a TAP service}i
+\section{The ivoa.obs\_radio table}
 \label{sec:implementation}
 
-The ObsCore extension for radio (including or not visibility data) described above SHOULD  be added to the main ObsCore table. The standardID for this extented table will be ivo://ivoa.net/std/ObsCore\#radioExt-1.0". This implies that the the core ObsCore attributes follow version 1.1 of the Obscore standard.
-\textit{ In practice a  table containing only the extension attributes  MAY be added to the same schema. The full extension table will then be a view joining the core table and the  table with extension attribue only via an extended ObsTAP ADQL query. A single dataset in each observation will be associated to a single row in ObsCore. It will be identified by a unique obs\_publisher\_did. This obs\_publisher\_did can be used as a foreign key to join the main table and the extension table.}
+The ObsCore Extension for Radio is accessed as a table within a TAP
+\citep{2019ivoa.spec.0927D} service.\todo{I think we should require a
+few UDFs on TAP services: ivo\_interval\_overlaps, ivo\_specconv}  At this
+point, the name of this table is fixed to \verb|ivoa.obs_radio|.
+However, in order to allow for later evolution -- which may allow
+multiple compliant tables per service --, this name should not be used
+for discovery (see sect.~\ref{sec:registry}).
+
+A TAP service that has \verb|ivoa.obs_radio| must also have a table
+compliant to any version 1 of ObsCore, i.e., a table
+\verb|ivoa.obscore|; the two tables must share exactly the columns such
+that a \verb|NATURAL JOIN| will yield per-dataset rows of obscore and
+radio extension metadata.  Which columns these are is site-specific.
+
+To ensure that all compliant services can execute the same queries,
+all columns in table~\ref{tab:ExtensionAtt} must be present in such a
+table, although any may be NULL.  Additional columns may be present, and
+at least a foreign key into \verb|ivoa.obscore| will typcially
+user-visible\todo{can we make rules such that such additional columns
+will not interfere with later extensions?}.
+
+The intention is that clients will write queries like
+\begin{lstlisting}
+SELECT [any obscore and obs_radio columns or expressions]
+FROM
+  ivoa.obscore
+  NATURAL JOIN ivoa.obs_radio
+WHERE
+  [constraints]
+\end{lstlisting}
+without having to know the exact foreign key(s) in use on any given
+service.  This means that while one service can opt to join on
+\verb|obs_publisher_did| (we expect this to be a very common choice),
+other services may choose to join on, say, artificial keys.
```

Comment:
Suggestion : 

few UDFs on TAP services: ivo\_interval\_overlaps, ivo\_specconv}  At this
point, the name of this table is fixed to \verb|ivoa.obs_radio|.
However, in order to allow for later evolution -- which may allow
multiple compliant tables per service --, this name should not be used
for discovery (see sect.~\ref{sec:registry}).

A TAP service that has \verb|ivoa.obs_radio| must also have a table
compliant to any version 1 of ObsCore, i.e., a table
\verb|ivoa.obscore|; the two tables must share exactly the columns such
that a \verb|NATURAL JOIN| will yield per-dataset rows of obscore and
radio extension metadata.  Which columns these are is site-specific.

To ensure that all compliant services can execute the same queries,
all columns in table~\ref{tab:ExtensionAtt} must be present in such a
table, although any may be NULL.  Additional columns may be present, and
at least a foreign key into \verb|ivoa.obscore| will typcially
user-visible\todo{can we make rules such that such additional columns
will not interfere with later extensions?}.

The intention is that clients will write queries like
\begin{lstlisting}
SELECT [any obscore and obs_radio columns or expressions]
FROM
  ivoa.obscore
  NATURAL JOIN ivoa.obs_radio
WHERE
  [constraints]
\end{lstlisting}
without having to know the exact foreign key(s) in use on any given
service.  This means that while one service can opt to join on
\verb|obs_publisher_did| (we expect this to be a very common choice),
other services may choose to join on, say, artificial keys.

Reply:
It's a bit hard to pick out your suggestion here...

You should be able to push into this branch -- so, can you just apply your proposed changes and merge?  If I really do not like them, I can still do new PRs against the new main branch.